config/routes.rb

```diff
@@ -1,10 +1,6 @@
-require "sidekiq/web"
-require 'sidekiq/cron/web'
 require "user_constraint"
 
 Rails.application.routes.draw do
-  mount Sidekiq::Web => "/sidekiq", constraints: UserConstraint.new
```

Comment:
Interesting :-) Let me five that a try (Not that keen on using master though, will fork for now)